contracts/chain-adapters/ZkStack_CustomGasToken_Adapter.sol

```diff
@@ -201,8 +215,28 @@ contract ZkStack_CustomGasToken_Adapter is AdapterInterface {
                     refundRecipient: L2_REFUND_ADDRESS
                 })
             );
+        } else if (l1Token == address(usdcToken)) {
+            if (_isCCTPEnabled()) {
+                _transferUsdc(to, amount);
+            } else {
+                IERC20(CUSTOM_GAS_TOKEN).forceApprove(USDC_SHARED_BRIDGE, txBaseCost);
+                IERC20(l1Token).forceApprove(USDC_SHARED_BRIDGE, amount);
+                txHash = BRIDGE_HUB.requestL2TransactionTwoBridges(
```

Comment:
nit: can probably refactor this request call with the below one into an internal function, where the input is the "bridge" contract to call `requestL2TransactionTwoBridges` on. This way there's less reused code